admin/InitDb.pl

```diff
@@ -308,6 +308,9 @@ sub CreateRelations
     RunSQLScript($DB, 'caa/CreateFKConstraints.sql', 'Adding CAA foreign key constraints ...')
         unless $REPTYPE == RT_MIRROR;
 
+    RunSQLScript($DB, 'documentation/CreateFKConstraints.sql', 'Adding documentation foreign key constraints ...')
+        unless $REPTYPE == RT_MIRROR;
```

Comment:
Is this change (in commit https://github.com/metabrainz/musicbrainz-server/pull/2523/commits/1324695b0188ec760a5274457c9339577267a048) following an earlier change or is it just something you used to do manually?

Reply:
It's not directly related to any other schema change in this release, but it was revealed due to the addition of new documentation FK constraints for the genre/mood l_* tables.